codespell_lib/data/dictionary.txt

```diff
@@ -10188,6 +10188,7 @@ ehance->enhance
 ehanced->enhanced
 ehancement->enhancement
 ehancements->enhancements
+ehen->when
```

Comment:
Suggest hen too as a typo.

Add ehenever as another typo too perhaps?

Reply:
Could also suggest ehen->even ?

Reply:
Added in bb06c83